_sass/_basic.scss

```diff
@@ -281,3 +281,7 @@ ul > li {
     margin-top: 3em;
     margin-bottom: 3em;
 }
+
+.pronouns {
+    font-size: medium;
```

Comment:
Just to add in here: things like `medium` are not great because they tend to obfuscate things from the reader of the code (in this case, `medium` is browser-defined IIRC). You may want to use a `em` measure instead, or as Leo mentioned, put this into a variable.

Reply:
> Hey Darren! Thanks for tackling this card. Things look good, though I think a variable might be in order for the font size of our pronouns.
> 
> For the blog post business, I think it'd look best if we put them in a slightly smaller font past the underline on names in the "Authors" section:
> ![image](https://user-images.githubusercontent.com/7704542/97647211-e17f1b00-1a0e-11eb-8edf-dde1f190c2ca.png)
> 
> These are both super nitpicky comments though, other than that this looks good imo.

I added a variable to the variables sass file for the pronouns font size, not 100% sure if this is what you meant. Also, for some reason I can't find the "Authors" section on the website.

Reply:
Great, the variable is exactly what we meant, so that's good! The "Authors" section of the website is slightly complicated, but the actual author card is rendered by `_includes/blog-author-card.html` - however, to pass it the correct pronoun data, you'll also need to adjust line 66 in `_layouts/post.html`. If that's confusing, feel free to either punt it to the next PR, or come to our OH to talk about it!